### PR TITLE
LRQA-164359 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-164359

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro [hunk 1]
```diff
@@ -88,7 +88,11 @@ definition {
 		}
 
 		if ("${exportPermissions}" == "true") {
-			Check(locator1 = "Checkbox#PERMISSIONS");
+			WaitForPageLoad();
+
+			AssertElementPresent(locator1 = "Checkbox#PERMISSIONS");
+
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro:94]: @melodywu We should always make our macros simple before adding any wait assertions. We'll reserve those as a last resort when our tests end up being flaky.

**Review by patriciajeanperez — COMMENTED** [review]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/124481

File: portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro (hunk 1 above)
Comment (patriciajeanperez) [reply portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro:94]: I ran the tests a couple of times in https://github.com/patriciajeanperez/liferay-portal/pull/660 and they seem to pass without the other assertions

Comment (patriciajeanperez) [reply portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro:94]: cc @sharryxue

Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/LAR.macro:94]: Ok thanks a lot

**Pull request closed without merging**